summernote/summernote
# Add nofollow attribute to link

#### What does this PR do?

- add/remove rel="nofollow" to link

#### How should this be manually tested?

- click "link" button
- check/uncheck: Add rel="nofollow" to link

#### Screenshots (if for frontend)

![screen shot 2016-02-20 at 3 05 35 pm](https://cloud.githubusercontent.com/assets/3636406/13195092/8b8fe994-d7e3-11e5-8888-d7dbc38ce38f.png)
